vcd/resource_vcd_org_vdc_test.go

```diff
@@ -34,8 +34,19 @@ func TestAccVcdOrgVdcReservationPool(t *testing.T) {
 		"Tags":                      "vdc",
 		"FuncName":                  "TestAccVcdOrgVdcReservationPool",
 		// cause vDC ignores empty values and use default
-		"MemoryGuaranteed": "1",
-		"CpuGuaranteed":    "1",
+		"MemoryGuaranteed":                   "1",
+		"CpuGuaranteed":                      "1",
+		"FlexElasticKey":                     "",
+		"FlexElasticValue":                   "",
+		"FlexElasticValueUpdate":             "",
+		"FlexMemoryOverheadKey":              "",
+		"equalsChar":                         "",
+		"FlexMemoryOverheadValue":            "",
+		"FlexMemoryOverheadValueUpdate":      "",
+		"ElasticityValueForAssert":           "false",
+		"ElasticityUpdateValueForAssert":     "false",
+		"MemoryOverheadValueForAssert":       "true",
+		"MemoryOverheadUpdateValueForAssert": "true",
```

Comment:
There are **four** additions of these **eleven** Flex-related test parameters in this PR. However, only **one** of them (in `TestAccVcdOrgVdcAllocationFlex`) actually sets these eleven parameters to non-empty:

![Screenshot 2020-02-11 at 15 32 02](https://user-images.githubusercontent.com/11330689/74240970-ac496600-4ce3-11ea-8fcb-b5ac9b6ce0d2.png)

I realize this has been done to reuse existing templates `testAccCheckVcdVdc_basic` and `testAccCheckVcdVdc_update`, but can this be factored more neatly? Or at least some visual grouping added by using comments, which also explain why these are here, so you don't need to read the whole file to understand what's happening.

Reply:
Improved

Reply:
Thanks!